lib/lib-table-cluster/src/test/java/org/sagebionetworks/table/cluster/MySqlColumnTypeTest.java

```diff
@@ -28,9 +26,9 @@ public void testParserTypeNoSize(){
 	
 	@Test
 	public void testParseSizeBigInt(){
-		String typeString = "bigint(20)";
+		String typeString = "BIGINT";
 		Integer size = MySqlColumnType.parseSize(typeString);
-		assertEquals(new Integer(20), size);
+		assertNull(size);

```

Comment:
add test case for VARCHAR of the parseSize() method. 

String typeString = "VARCHAR(56)";